Grand.Web/Areas/Admin/Views/Product/_CreateOrUpdate.Info.cshtml

```diff
@@ -46,6 +47,7 @@
         toggleCustomerEntersPrice();
         toggleBasepriceEnabled();
         toggleDownloadableProduct();
+        toggleSizeChartPicture;
```

Comment:
Hi, Just wondering why we are not calling this function, just like others. Maybe **()** is missing.

Reply:
Very good question. It looks like that was a duplicate line (almost) - there was another toggle a few lines down. There were a couple of other fixes missing from that last commit too. I just added them.